### ISSUE TITLE
SearchBox: borders not visible in high contrast

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#787]
![image](https://cloud.githubusercontent.com/assets/11937034/21847061/ee6ab3b8-d81f-11e6-9f84-9e80f0f63ce6.png)

The screen shot is taken from the fabric-react demo page.

**Comment by prmittal** [comment]
@dzearing Is there any update on this?

**Comment by prmittal** [comment]
@battletoilet Is there an ETA for this?

**Comment by prmittal** [comment]
@micahgodbolt Is there any update here?

**Comment by stephenzsy** [comment]
We are blocked by this issue

**Comment by mdahamiwal** [comment]
@prmittal, @stephenzsy, this is fixed in latest official:

![image](https://cloud.githubusercontent.com/assets/12109362/25837750/de93c3b4-34ac-11e7-892c-7a0042eb8a15.png)

**Comment by betrue-final-final** [comment]
Is the last screenshot the change? If so, that appears to be incorrect. A good rule of thumb is "equality". Make it look like it does on non-high contrast, with high contrast colors.

Doesn't this just have a bottom border?
